package protoFile.hall.InviteJoinGroupAccept;
message InviteJoinGroupAcceptRequest{
     required int32 id=1;//角色id
     required int32 inviteId = 2;//邀请人id
     required int32 areaId=3;//房间所在区域的id
     required int32 roomId=4;//邀请的房间id
}
message InviteJoinGroupAcceptResponse{
     required bool result=1;
     optional string message=2;
}